### PR TITLE
Adding MCG and OCP Operator

### PR DESCRIPTION
**Pull request by apinnick**
DO NOT MERGE. This PR is on hold, pending release of CAM 1.1.0 or 1.1.1.

### CODE REVIEW

File: migration/migrating-3-4/migrating-openshift-3-to-4.adoc [hunk 1]
```diff
@@ -23,17 +23,17 @@ Before you begin your migration, be sure to review the information on xref:../..
 .Prerequisites
 
 * The source cluster must be {product-title} 3.7, 3.9, 3.10, or 3.11.
-* The target cluster must be {product-title} 4.2.
+* The target cluster must be {product-title} {product-version}.
 * You must have xref:../../cli_reference/openshift_cli/administrator-cli-commands.html#policy[`cluster-admin` privileges] on all clusters.
 * You must have `podman` installed.
-* You must configure a replication repository that supports the S3 API and is accessible to the source and target clusters.
+* The cluster on which the CAM tool is installed must have a 50 GB PVC for the MCG replication repository.
```
Comment (jwmatthews) [inline migration/migrating-3-4/migrating-openshift-3-to-4.adoc]: We are looking more at confirming this.

@jmontleon please update as we learn what the default requirements are.

File: migration/migrating-3-4/migrating-openshift-3-to-4.adoc [hunk 2]
```diff
@@ -54,21 +54,21 @@ include::modules/migration-understanding-cam.adoc[leveloffset=+1]
...
+The CAM tool uses the replication repository as a temporary intermediate storage space for copying data from the source cluster to the target cluster, using the xref:migration-understanding-data-copy-methods_{context}[file system or snapshot copy method].
 
 The following storage providers are supported:
 
-// * xref:migration-configuring-noobaa_{context}[NooBaa] or other generic S3 storage provider (does not support snapshot copy method)
-* Multi Cloud Gateway (MCG) or other generic S3 storage provider
+* MCG or other generic S3 storage provider
 +
-MCG is installed by the CAM Operator and does not support the snapshot copy method.
+MCG, which is installed automatically on the same cluster as the CAM tool, does not support the snapshot copy method.
```
Comment (jwmatthews) [inline migration/migrating-3-4/migrating-openshift-3-to-4.adoc]: Please remove the text on 'does not support the snapshot copy method.'
Snapshots are a feature of services providing volumes, block devices, etc, not for object storage.

A user who is using MCG for Object Store could still perform a snapshot on their PVs if the PV backing store supported snapshots.

but MCG/S3, etc is not involved in the snapshot piece.

File: migration/migrating-4-4/migrating-openshift-4_1-to-4.adoc [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
-You can migrate application workloads from {product-title} 4.1 to 4.2 with the Cluster Application Migration (CAM) tool. The CAM tool enables you to control the migration and to minimize application downtime.
+You can migrate application workloads from {product-title} 4.1 to {product-version} with the Cluster Application Migration (CAM) tool. The CAM tool enables you to control the migration and to minimize application downtime.
 
 The CAM tool's web console and API, based on Kubernetes custom resources, enable you to migrate stateful and stateless application workloads at the granularity of a namespace.
 
 .Prerequisites
 
 * You must have xref:../../cli_reference/openshift_cli/administrator-cli-commands.html#policy[`cluster-admin` privileges] on all clusters.
-* You must configure a replication repository that supports the S3 API and is accessible to the source and target clusters.
+* The cluster on which the CAM tool is installed must have a 50 GB PVC for the MCG replication repository.
```
Comment (jwmatthews) [inline migration/migrating-4-4/migrating-openshift-4_1-to-4.adoc]: Similar to 3.x side, we are confirming what the default requirement is.

File: migration/migrating-4-4/migrating-openshift-4_1-to-4.adoc [hunk 4]
```diff
@@ -41,21 +41,21 @@ include::modules/migration-understanding-cam.adoc[leveloffset=+1]
...
+The CAM tool uses the replication repository as a temporary intermediate storage space for copying data from the source cluster to the target cluster, using the xref:migration-understanding-data-copy-methods_{context}[file system or snapshot copy method].
 
 The following storage providers are supported:
 
-// * xref:migration-configuring-noobaa_{context}[NooBaa] or other generic S3 storage provider (does not support snapshot copy method)
-* Multi Cloud Gateway (MCG) or other generic S3 storage provider
+* MCG or other generic S3 storage provider
 +
-MCG is installed by the CAM Operator and does not support the snapshot copy method.
+MCG does not support the snapshot copy method.
```
Comment (jwmatthews) [inline migration/migrating-4-4/migrating-openshift-4_1-to-4.adoc]: Please remove this text

File: migration/migrating-4-4/migrating-openshift-4_1-to-4.adoc [hunk 5]
```diff
@@ -76,7 +76,7 @@ include::modules/migration-installing-cam-operator-ocp-4.adoc[leveloffset=+2]
 :context: targetcluster-4_1-4_2
 :targetcluster-4_1-4_2:
 include::modules/migration-installing-cam-operator-ocp-4.adoc[leveloffset=+2]
-:targetcluster-4_1-4_2!:
+:targetcluster-4_1-4_2!:* The cluster on which the CAM tool is installed must have a 50 GB PVC for the MCG replication repository.
```
Comment (jwmatthews) [inline migration/migrating-4-4/migrating-openshift-4_1-to-4.adoc]: Verifying the actual requirement

File: modules/migration-installing-cam-operator-ocp-4.adoc [hunk 6]
```diff
@@ -49,7 +50,7 @@ ifdef::sourcecluster-4_1-4_2[]
 . Click *Catalog* -> *OperatorHub*.
 endif::[]
 . On the *OperatorHub* page:
-.. Scroll or type a keyword into the *Filter by keyword* field (in this case, *Migration*) to find the *Cluster Application Migration Operator*.
+.. Scroll or type a keyword into the *Filter by keyword* field (in this case, *Migration*) to find the *CAM Operator*.
```
Comment (jwmatthews) [inline modules/migration-installing-cam-operator-ocp-4.adoc]: I'm not aware of any name change to 'CAM Operator', it should be 'Cluster Application Migration Operator'.

**Review by jwmatthews — CHANGES_REQUESTED** [review]

**Comment by jwmatthews** [comment]
@apinnick please keep this PR on hold and do _not_ merge it.

The support for MCG/NooBaa will not be part of our 1.0.1 release.   
We will likely bring it in with 1.1.0 or perhaps 1.1.1, it's not ready for what we need in 1.0.1.

**Comment by apinnick** [comment]
> @apinnick please keep this PR on hold and do _not_ merge it.
> 
> The support for MCG/NooBaa will not be part of our 1.0.1 release.
> We will likely bring it in with 1.1.0 or perhaps 1.1.1, it's not ready for what we need in 1.0.1.

@jwmatthews Understood. I will make the changes and keep this PR as 'draft PR' so that it won't be merged.

**Comment by apinnick** [comment]
Closing this PR and opening new one

**Pull request closed without merging**